libs/etcd_broker/src/lib.rs

```diff
@@ -383,7 +388,8 @@ mod tests {
                 &timeline_subscription,
             ] {
                 let (id, _timeline) =
-                    parse_etcd_key_value(subscription, &key_string, value_str)
+                    parse_safekeeper_timeline_kv(subscription, &key_string, value_str)
+                        .unwrap()
```

Comment:
I don't feel good about that `.unwrap`, given that we cannot control what we receive: let's rewrite it so that there's no way unusual KV input can cause panics here.